serverless/src/util/jsonToCsv.js

```diff
@@ -0,0 +1,26 @@
+import { get } from 'sendero'
+import { unparse } from 'papaparse'
+
+/**
+ * converts an array of JSON objects to a CSV
+ * @param {Array} jsonArray array of JSON objects
+ * @param {Array} columns - array of column objects with name and dot path to value, such as [{ name: "Platform", path: "platforms.shortName" }, ...]
+ */
+export const jsonToCsv = (jsonArray, columns) => {
+  const data = jsonArray.map((item) => (
+    columns.reduce((row, { name, path }) => ({
+      ...row,
+      [name]: get(item, path, {
+        clean: true, sort: true, stringify: true, unique: true
+      }).join(', ')
+    }), {})
+  ))
+
+  const config = {
+    columns: columns.map(({ name }) => name)
+  }
+
+  // adding line break at end to ensure backwards compatability
+  // but may remove it in the future
```

Comment:
If we have a plan to remove it, I would make a ticket and reference that here. Otherwise, this comment will definitely get lost.

Reply:
Good point.  I think we can leave it as is.  As long the current way works, not sure there would be a need to change it.  I updated via https://github.com/nasa/earthdata-search/pull/1630/commits/8fd6401e93e6c0a3f72ac355433c1e18f531f328